src/commands/run.rs

```diff
@@ -228,16 +255,31 @@ pub struct RunCommand {
     trap_on_grow_failure: bool,
 }
 
+enum Profile {
+    Native(wasmtime::ProfilingStrategy),
```

Comment:
Care to (in a follow up) rename `wasmtime::ProfilingStrategy` to `wasmtime::NativeProfilingStrategy` and have

```rust
#[deprecated("Renamed to NativeProfilingStrategy")]
pub type ProfilingStrategy = NativeProfilingStrategy;
```

for backwards compatibility?

Reply:
Sure!